specification/communication/data-plane/Email/preview/2023-15-01-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,413 @@
+            "description": "Message payload for sending an email",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Message"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Message accepted, the `Operation-Location` header contains the URL to poll for the message status.",
+            "headers": {
+              "Repeatability-Result": {
+                "description": "Status of a repeatable request.",
+                "type": "string"
+              },
+              "Operation-Location": {
+                "description": "Location url of where to poll the status of this message from.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Amount of time client should wait before retrying the request, specified in seconds ",
+                "type": "integer"
+              },
+              "x-ms-request-id": {
+                "description": "System generated GUID indicating the id of this specific message, to be used for tracking purposes.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Send Email": {
+            "$ref": "./examples/SendEmail.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Message": {
+      "description": "Message payload for sending an email",
+      "required": [
+        "senderEmail",
+        "content",
+        "recipients"
+      ],
+      "type": "object",
+      "properties": {
+        "headers": {
+          "description": "Custom email headers to be passed.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "senderEmail": {
+          "description": "Sender email address from a verified domain.",
+          "type": "string",
+          "example": "info@contoso.com"
+        },
+        "content": {
+          "description": "Email content to be sent.",
+          "$ref": "#/definitions/EmailContent"
+        },
+        "importance": {
+          "description": "The importance type for the email.",
+          "$ref": "#/definitions/EmailImportance"
+        },
+        "recipients": {
+          "description": "Recipients for the email.",
+          "$ref": "#/definitions/EmailRecipients"
+        },
+        "attachments": {
+          "description": "list of attachments",
```

Comment:
Are there any service restrictions that need to be called out, e.g. max number of attachments or max file size?

Reply:
We enforce a limit on the total email size (not specifically for attachments). Anyway, wouldn't it make more sense to mention this in documentation than here (reason being if we want to update this in the future, we can just update the documentation, instead of having to update the swagger file)?

Reply:
@apattath The API reference documentation gets generated from the swagger, so by adding it here you add it to the documentation, and if it's missing it's missing from the documentation :-)

Reply:
Updated in the next iteration.